app/client/src/components/pages/Community/components/routes/CommunityIntro.js

```diff
@@ -93,13 +93,23 @@ function CommunityIntro({ ...props }: Props) {
 
         <Topic>
           <TopicIcon>
-            <img src={fishingIcon} alt="Fishing" />
+            <img src={fishingIcon} alt="Eating Fish" />
           </TopicIcon>
           <TopicText>
-            <strong>Fish Consumption and Aquatic Life:</strong> &nbsp; EPA,
-            states, and tribes monitor and assess water quality to determine the
-            impact of impairments on plants and animals living in the water.
-            They also monitor and assess if fish and shellfish are safe to eat.
+            <strong>Fish Consumption:</strong> &nbsp; EPA, states, and tribes
```

Comment:
This should be changed to Eating Fish.

```suggestion
            <strong>Eating Fish:</strong> &nbsp; EPA, states, and tribes
```

Reply:
Thanks! I noticed a place on the data page where we listed out all of the tabs and I've updated that too.